app/scss/questionPage.scss

```diff
@@ -312,3 +312,7 @@ img {
   border: 1px solid $question-nav-border;
   padding: 7px 0 31px;
 }
+
+.visibilityModal {
```

Comment:
visibilityModal is a strange name, not very semantic (i'm thinking what's a visibility modal?), can you change it to something more semantic?

oh actually I don't think this class is used anywhere? if it isn't then remove it